docs/uppmax.md

```diff
@@ -74,14 +76,170 @@ It is not suitable for use with real data.
+If you use `rackham` to download the pipeline and the singularity containers, we recommend using an interactive session (cf [interactive guide](https://www.uppmax.uu.se/support/faq/running-jobs-faq/how-can-i-run-interactively-on-a-compute-node/)), which is what we do in the following guide.
+
+### Download and install Nextflow
+
+You can use the Nextflow UPPMAX provided `module`, but if necessary, you can also download a more recent version.
+
+```bash
+# Connect to rackham
+$ ssh -X <USER>@rackham.uppmax.uu.se
+# Or stay in your terminal
+
+# Download the nextflow-all bundle
+$ wget https://github.com/nextflow-io/nextflow/releases/download/v<NEXTFLOW_VERSION>/nextflow-<NEXTFLOW_VERSION>-all
+
+# Connect to the wharf area using sftp
+$ sftp <USER>-<BIANCA_PROJECT>@bianca-sftp.uppmax.uu.se:<USER>-<BIANCA_PROJECT>
+
+# Transfer nextflow to the wharf area
+sftp> put nextflow-<NEXTFLOW_VERSION>-all .
+
+# Exit sftp
+$ exit
+
+# Connect to bianca
+$ ssh -A <USER>-<BIANCA_PROJECT>@bianca.uppmax.uu.se
+
+# Go to your project
+$ cd /castor/project/proj_nobackup
+
+# Make folder for Nextflow
+$ mkdir tools
+$ mkdir tools/nextflow
+
+# Move Nextflow from the wharf area to its directory
+$ mv /castor/project/proj_nobackup/wharf/<USER>/<USER>-<BIANCA_PROJECT>/nextflow-<NEXTFLOW_VERSION>-all /castor/project/proj_nobackup/tools/nextflow
 
-For security reasons, there is no internet access on Bianca so you can't download from or upload files to the cluster directly. Before running a nf-core pipeline on Bianca you will first have to download the pipeline and singularity images needed elsewhere and transfer them via the wharf area to your Bianca project.
+# Establish permission
+$ chmod a+x /castor/project/proj_nobackup/tools/nextflow/nextflow-<NEXTFLOW_VERSION>-all
 
-You can follow the guide for downloading pipelines [for offline use](https://nf-co.re/tools#downloading-pipelines-for-offline-use). Note that you will have to download the singularity images as well.
+# If you want other people to use it
+# Be sure that your group has rights to the directory as well
+$ chown -R .<BIANCA_PROJECT> /castor/project/proj_nobackup/tools/nextflow/nextflow-<NEXTFLOW_VERSION>-all
+
+# Make a link to it
+$ ln -s /castor/project/proj_nobackup/tools/nextflow/nextflow-<NEXTFLOW_VERSION>-all /castor/project/proj_nobackup/tools/nextflow/nextflow
+
+# And every time you're launching Nextflow, don't forget to export the following ENV variables
+# Or add them to your .bashrc file
+$ export NXF_HOME=/castor/project/proj/nobackup/tools/nextflow/
+$ export PATH=${NXF_HOME}:${PATH}
+$ export NXF_TEMP=$SNIC_TMP
+$ export NXF_LAUNCHER=$SNIC_TMP
+$ export NXF_SINGULARITY_CACHEDIR=/castor/project/proj_nobackup/singularity-images
+```
+
+### Install nf-core tools
+
+```bash
+# Connect to rackham
+$ ssh -X <USER>@rackham.uppmax.uu.se
+# Or stay in your terminal
+
+# Install the latest pip version
+$ pip3 install --upgrade --force-reinstall git+https://github.com/nf-core/tools.git@dev --user
```

Comment:
We should not be recommending that people use the dev version. It's very rare that this is necessary.

Reply:
Also this should be available as a module now I think. So can just do `module load nf-core`

Reply:
I used the `dev` version because I wanted do download a DSL2 pipeline, the version available on module seems a bit old.

Reply:
I'd also say pip3-installing stuff is fairly brittle/risk breaking other stuff, so I'd much prefer referring to a module or a container if possible.

Reply:
I suggest using this for now. Once there is a tools release with container support I can commit to creating a PR for that (probably also moving to using transfer).